purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1372,7 +1407,33 @@ class Purchases internal constructor(
         }
     }
 
-    private fun getPurchaseCallback(productId: String): PurchaseCallback? {
+    internal fun purchaseNonUpgradeWithDeprecatedCallback(
```

Comment:
could probably do something like [toPurchaseCallback](https://github.com/RevenueCat/purchases-android/blob/803eb7520d61d181828c03b8fb079a576c0fbffb/purchases/src/main/kotlin/com/revenuecat/purchases/interfaces/PurchaseCallback.kt#L19) did in the last release?

Reply:
I love this name, TBH 🙃